### PR TITLE
FIR-32015: implemented missing trivial setters in PreparedStatement

### PR DESCRIPTION
**Pull request by alexradzin**
setURL
setByte

### CODE REVIEW

**Review by ptiurin — COMMENTED** [review]
Also, in general, do we even have BYTE data type? AFAIK there's only BYTEA, which is an array of bytes. Should `setByte` populate a single byte BYTEA type? Can we test this?

File: src/test/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatementTest.java
[the remark this answers is not in the document]
Comment (alexradzin) [reply src/test/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatementTest.java:1]: test is added

[the remark this answers is not in the document]
Comment (ptiurin) [reply src/test/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatementTest.java:1]: What about other types apart from URL? How do they behave when set to null?

**Comment by alexradzin** [comment]
> ingle byte BYTEA type? Can we tes

Byte is a very small int. So, I do not see any reason not to support it on the JDBC level

File: src/test/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatementTest.java
[the remark this answers is not in the document]
Comment (alexradzin) [reply src/test/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatementTest.java:1]: Should be OK. In most cases it is implemented in `JavaTypeToFireboltSQLString.transform(Object object)`:

```java
public String transform(Object object) throws FireboltException {
    if (object == null) {
	    return NULL_VALUE;
    } else {
	    try {
		    return transformToJavaTypeFunction.apply(object);
	    } catch (Exception e) {
		    throw new FireboltException("Could not convert object to a String ", e, TYPE_TRANSFORMATION_ERROR);
	    }
    }
}
```

**Comment by alexradzin** [comment]
added  integration tests

**Comment by ptiurin** [comment]
> added integration tests

Let's also include URL there

**Comment by alexradzin** [comment]
added integration test for getURL()/setURL()

**Review by ptiurin — APPROVED** [review]

**Pull request merged**